ouroboros-consensus/src/unstable-consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/Adversarial.hs

```diff
@@ -484,8 +515,11 @@ uniformAdversarialChain mbAsc recipe g0 = wrap $ C.createV $ do
             maybe (RI.initConservative vA) id
           $ RI.init kcp vA
 
-    ensureLowerDensityThanHonestSchema g mv
+    ensureLowerDensityInWindows iterH g mv
```

Comment:
Now that your other function declares `iterH = ... vH`, I'm wondering if the race iterator here (declared as `iterH` just a few lines up) should be named `iterA` instead, since it's iterating over the race windows based on the adversarial schema. What do you think?